src/module.c

```diff
@@ -6119,8 +6121,8 @@ int RM_DictCompare(RedisModuleDictIter *di, const char *op, RedisModuleString *k
 int RM_InfoEndDictField(RedisModuleInfoCtx *ctx);
 
 /* Used to start a new section, before adding any fields. the section name will
- * be prefixed by "<modulename>_" and must only include A-Z,a-z,0-9.
- * NULL or empty string indicates the default section (only <modulename>) is used.
+ * be prefixed by "\<modulename\>\_" and must only include A-Z,a-z,0-9.
```

Comment:
same here, i would love to find a way to handle that without the escaping in the C code.
maybe using `?

Reply:
Yeah, anything enclosed in backticks is left as-is. I'll do that.

You can test here in Github comments preview tab how various markdown is rendered, e.g. `"<modulename>_"` :-)